spec/webview-spec.js

```diff
@@ -840,12 +828,6 @@ describe('<webview> tag', function () {
 
   describe('executeJavaScript', () => {
     it('should support user gesture', function (done) {
-      if (process.env.TRAVIS !== 'true' || process.platform === 'darwin') {
```

Comment:
It looks like this test was previously skipped on Mac, but it turns out it passes, cool!

Reply:
@alexeykuzmin yeah... I noticed that this test basically wasn't running anywhere (except Travis Linux), so I enable to see if it would pass.